src/components/Symptoms/Symptoms.js

```diff
@@ -0,0 +1,28 @@
+import Fieldset from "../Fieldset/Fieldset";
+
+function Symptoms({ handleChange, points }) {
+  const inputsInfo = [
```

Comment:
Zmieniłabym nazwę na bardziej związaną z tym co zawiera. To są objawy, więc coś w tym tylu:
```suggestion
  const symptoms = [
```

Reply:
done